requirements.txt

```diff
@@ -4,4 +4,10 @@ deprecation>=2.1.0
 pytest>=2.7.2
 PyYAML>=5.1
 rdflib>=6.0.0
-pre-commit>=2.19.0
+pre-commit
+pydocstyle
```

Comment:
Remove pydocstyle, it will be installed indirectly with flake8

Reply:
The task was to align the proxy settings, I don't want to mix the stuff here.